frontend/src/components/partials/MapView.js

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from "react";
+
+class MapView extends Component {
```

Comment:
So for components which don't have lifecycle methods and state you can make them stateless